deploy/group_vars/all.yml

```diff
@@ -77,7 +92,7 @@ k8s_iam_users: [copelco]
 # Pin ingress-nginx and cert-manager to current versions so future upgrades of this
 # role will not upgrade these charts without your intervention:
 # https://github.com/kubernetes/ingress-nginx/releases
-k8s_ingress_nginx_chart_version: "4.0.19"
+k8s_ingress_nginx_chart_version: "4.4.2"
```

Comment:
Can you add the helm list output for ingress-nginx and cert-manager?

Reply:
Done, it's in **Upgrade ingress-nginx and cert-manager** section.